### ISSUE TITLE
looking for a script can anyone help

### ISSUE THREAD

**Issue by paigemaster** [kolton/d2bot-with-kolbot#249]
im looking for a seal basher script one that will hostile seal bashers anyone know where i can find one for kolbot???

**Comment by LeMule** [comment]
It shouldn't be too hard to make. I might have a looksee if I can slap
something together, weekend's on the way, just have to check that weather
first!
You should expect that your bots are going to get massacred once people
catch on to this, assuming you're public.

**Comment by paigemaster** [comment]
Sweet i saw this on that warned you about hitting seals then if u did it
again it would kill bashers I'm sure I'll prolly get raped a few times

**Comment by paigemaster** [comment]
can that be set to use with team botting?

**Comment by Legless** [comment]
@LeMule 
> It shouldn't be too hard to make.
How can you know who hitted the seal? 
Or the idea is to hostile everyone in game except your chars?

**Comment by paigemaster** [comment]
Well the one time that I saw it before the bat didn't hostel people that
were hiding so anybody that was hiding with safe

**Comment by paigemaster** [comment]
Let me rephrase the way I put that if I would hostel everybody but if you went and hid the but wouldn't say sue the bot Wautoma kill open players I hope that makes a little more sense


Sent from my Samsung Galaxy smartphone.
-------- Original message --------From: Gaetan Poirier <notifications@github.com> Date: 8/10/17  6:10 PM  (GMT-05:00) To: kolton/d2bot-with-kolbot <d2bot-with-kolbot@noreply.github.com> Cc: paigemaster <brisatay2003@gmail.com>, Author <author@noreply.github.com> Subject: Re: [kolton/d2bot-with-kolbot] looking for a script can anyone help
  (#249) 
It shouldn't be too hard to make. I might have a looksee if I can slap

something together, weekend's on the way, just have to check that weather

first!

You should expect that your bots are going to get massacred once people

catch on to this, assuming you're public.



On Thu, Aug 10, 2017 at 7:45 AM, paigemaster <notifications@github.com>

wrote:

**Comment by LeMule** [comment]
@Legless, yeah, just rip everyone, why not.
To be serious though I had not taken that into consideration.

I'm not sure if other player's locations, and whom they are associated
with, are available, but if so (and this is by far not a perfect solution),
one could deduce the culprit from this information.
Sounds like I'd possibly be getting myself into something I don't really
want to though. Not at this time anyways.

**Comment by paigemaster** [comment]
Well honestly it was pretty kool when I saw it I'm not quite sure I'm
understanding where your going do u not want to make it ? It's fine if u
don't it's prolly not like I'd use it much I just saw it and it was one of
them got to have you know
